GameJam24/Assets/Scripts/Movement/FallingState.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class FallingState : IMovementState
+{
+    private Playermovement _player;
+    private Rigidbody2D _rb;
+    private InputActionAsset _inputActions;
+    private InputAction _moveInputAction;
+    
+    public void Initialize(Playermovement player)
+    {
+        _player = player;
+        _rb = player.GetComponent<Rigidbody2D>();
+        _inputActions = player.inputActions;
+        _moveInputAction = _inputActions.FindAction("Move");
+    }
+
+    public void OnEnter()
+    {
+        
+    }
+
+    public void OnUpdate()
+    {
+        _rb.AddForce(new Vector2(_moveInputAction.ReadValue<float>(),0), ForceMode2D.Impulse);
```

Comment:
Maybe do this in OnFixedUpdate()? Time sensitive stuff like physics should be computed there.